sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/samples/SampleSnippets.cs

```diff
@@ -18,25 +18,25 @@ public void CreateDocumentIntelligenceClient()
             #region Snippet:CreateDocumentIntelligenceClient
 #if SNIPPET
             string endpoint = "<endpoint>";
-            string apiKey = "<apiKey>";
 #else
             string endpoint = TestEnvironment.Endpoint;
-            string apiKey = TestEnvironment.ApiKey;
 #endif
-            var client = new DocumentIntelligenceClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            var client = new DocumentIntelligenceClient(new Uri(endpoint), new DefaultAzureCredential());
```

Comment:
Shouldn't all of these use `TestEnvironment.Credential` when running and show DAC injected into samples?   Pretty sure DAC is going to fail under federated auth in the pipelines.

Reply:
That's a good point 🤦 Let me update it across the board.